src/OpenTelemetry.Api/Context/Propagation/ITextFormat.cs

```diff
@@ -45,10 +45,11 @@ public interface ITextFormat
         /// Extracts activity context from textual representation.
         /// </summary>
         /// <typeparam name="T">Type of object to extract context from. Typically HttpRequest or similar.</typeparam>
+        /// <param name="activityContext">Activity context to transmit over the wire.</param>
```

Comment:
comment is not so correct.. Extract is to "extract" context from the wire.

Reply:
good catch! i will update!